gateway-ha/src/main/java/io/trino/gateway/ha/module/RouterBaseModule.java

```diff
@@ -65,4 +69,13 @@ public QueryHistoryManager getQueryHistoryManager()
     {
         return this.queryHistoryManager;
     }
+
+    @Provides
+    @Singleton
+    public BackendsMetricStats getBackendsMetricStats(GatewayBackendManager gatewayBackendManager, MBeanExporter exporter, MonitorConfiguration monitorConfiguration)
+    {
+        BackendsMetricStats backendsMetricStats = new BackendsMetricStats(gatewayBackendManager, exporter, monitorConfiguration);
+        backendsMetricStats.initMetrics();
```

Comment:
If we call `initMetrics()` in the `@PostConstruct`, can we get rid of the need for this custom injector?